flexibleengine/acceptance/resource_flexibleengine_compute_volume_attach_v2_test.go

```diff
@@ -0,0 +1,247 @@
+						"flexibleengine_compute_instance_v2.instance_1", "id"),
+					resource.TestCheckResourceAttrPair(resourceName, "volume_id", "flexibleengine_blockstorage_volume_v2.test", "id"),
+					resource.TestCheckResourceAttr(resourceName, "device", "/dev/vdb"),
+					resource.TestCheckResourceAttrSet(resourceName, "pci_address"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccComputeVolumeAttach_multiple(t *testing.T) {
+	var va block_devices.VolumeAttachment
+	rName := acceptance.RandomAccResourceNameWithDash()
+	rc := acceptance.InitResourceCheck(
+		"flexibleengine_compute_volume_attach_v2.test",
+		&va,
+		getVolumeAttachResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeVolumeAttach_multiple(rName),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckMultiResourcesExists(2),
+					resource.TestCheckResourceAttrPair("flexibleengine_compute_volume_attach_v2.test.0", "instance_id",
+						"flexibleengine_compute_instance_v2.test.0", "id"),
+					resource.TestCheckResourceAttrPair("flexibleengine_compute_volume_attach_v2.test.0", "volume_id",
+						"flexibleengine_blockstorage_volume_v2.test", "id"),
+					resource.TestCheckResourceAttrPair("flexibleengine_compute_volume_attach_v2.test.1", "instance_id",
+						"flexibleengine_compute_instance_v2.test.1", "id"),
+					resource.TestCheckResourceAttrPair("flexibleengine_compute_volume_attach_v2.test.1", "volume_id",
+						"flexibleengine_blockstorage_volume_v2.test", "id"),
+				),
+			},
+		},
+	})
+}
+
+const testAccCompute_data = `
+data "flexibleengine_availability_zones" "test" {}
+
+data "flexibleengine_compute_flavors_v2" "test" {
+  availability_zone = data.flexibleengine_availability_zones.test.names[0]
+  performance_type  = "normal"
+  cpu_core          = 2
+  memory_size       = 4
+}
+
+data "flexibleengine_vpc_subnet_v1" "test" {
+  name = "subnet-default"
+}
+
+data "flexibleengine_images_image" "test" {
+  name        = "OBS Ubuntu 18.04"
+  most_recent = true
+}
+
+data "flexibleengine_networking_secgroup_v2" "test" {
+  name = "default"
+}
+`
+
+func testAccComputeVolumeAttach_basic(rName string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "flexibleengine_blockstorage_volume_v2" "test" {
+  name              = "%s"
+  availability_zone = data.flexibleengine_availability_zones.test.names[0]
+  volume_type       = "SAS"
+  size              = 10
+}
+
+resource "flexibleengine_compute_instance_v2" "instance_1" {
+  name               = "%s"
+  image_id           = data.flexibleengine_images_image.test.id
+  flavor_id          = data.flexibleengine_compute_flavors_v2.test.flavors[0]
+  security_groups    = [data.flexibleengine_networking_secgroup_v2.test.name]
+  availability_zone  = data.flexibleengine_availability_zones.test.names[0]
```

Comment:
There have two spaces of the left of equals symbol.

Please check other scripts in the UT, such as 198 and 229 lines